fossunited/www/book-conference-ticket/index.py

```diff
@@ -0,0 +1,193 @@
+import base64
+import json
+
+import frappe
+from frappe.integrations.utils import (
+    make_get_request,
+    make_post_request,
+)
+
+# getting conference name data
+
+
+def get_context(context):
+    context.events = frappe.get_all(
+        "FOSS Chapter Events",
+        filters={
+            "status": "approved",
+            "event_type": "CityFOSS Conference",
+        },
+        fields=["event_name", "event_type"],
+    )
+
+
+# Script Script for creating order ID
+
+
+@frappe.whitelist()
+def add_ticket_to_doc(ticketsData):
+    values = json.loads(ticketsData)
+
+    print(values)
+
+    # events = frappe.get_doc("Conference Tickets", values['events'])
+    student_tickets = values["student_tickets"]
+    general_tickets = values["general_tickets"]
+
```

Comment:
I feel we can create a field for tshirt price too, and set it default to zero? Calcualte the total amount based on it as well.

Reply:
we can do that after the release, shouldn't be a lot of work that time.

Reply:
Sure - but make it asap after release, as conferences, if any happening in April should have this, as we should host them in the platform.

Reply:
Sure